### PR TITLE
TOOLS-3107 Update CHANGELOG.md for 100.5.3 release

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

**Comment by autarch** [comment]
I recommend using [grip](https://github.com/joeyespo/grip) for viewing the file locally so you can see how it will look on GitHub.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 # Database Tools Changelog
 
+## 100.5.3
+
+_Released 2022-06-XX_
+
+We are pleased to announce version 100.5.3 of the MongoDB Database Tools.
+
+This release contains a number of bug fixes and changes. Highlights include support for clustered collections in mongorestore, updating our Go version to address CVEs, and suported platform updates.
```
Comment (tfogo) [inline CHANGELOG.md:9]: Let's specifically mention we upgraded from 1.16.7 to 1.17.8.

s/suported/supported.

**Review by tfogo — CHANGES_REQUESTED** [review]
Looking good, one suggestion and a typo.

File: CHANGELOG.md (hunk 1 above)
Comment (autarch) [reply CHANGELOG.md]: Fixed both.

**Review by tfogo — APPROVED** [review]
LGTM

**Pull request merged**